src/account/handle.rs

```diff
@@ -188,7 +188,15 @@ impl AccountHandle {
         Ok(outputs)
     }
 
-    /// Returns all transaction of the account
+    /// Returns all incoming transactions of the account
+    pub async fn list_incoming_transactions(
+        &self,
+    ) -> Result<Vec<(TransactionId, (TransactionPayload, Vec<OutputResponse>))>> {
```

Comment:
Just curious why a second tuple in the tuple?

Reply:
Make it look like a Hashmap?

Reply:
Hm yeah because it's one, probably better to also return it as one, just remembered that there was something with returning it in nodejs as hashmap, but that's also fine
Will change